src/model/semantics.ml

```diff
@@ -57,6 +70,24 @@ let handle_internal_tags (type a) tags : a handle_internal_tags -> a = function
           None
       | Some ((`Dot _ as p), _) -> Some p
       | None -> None)
+  | Expect_page_tags ->
+      let unparsed_lines =
+        find_tags []
+          (function `Children_order _ as p -> Some p | _ -> None)
```

Comment:
I think `find_tags` would be easier to read if `f` was labelled:
```suggestion
          ~filter:(function `Children_order _ as p -> Some p | _ -> None)
```

Reply:
Now `find_tags` is inconsistent with `find_tag` immediately above it, which still takes an `f` parameter :-P

Reply:
Thanks!